lib/src/theme/typography.dart

```diff
@@ -31,69 +35,102 @@ class DSFRTypography extends ThemeExtension<DSFRTypography> {
     required this.boldText,
     required this.badgeLabel,
     required this.badgeLabelSmall,
+    required this.frLabel,
+    required this.frFieldsetLegend,
+    required this.frHintText,
+    required this.frErrorText,
```

Comment:
I wanted to write generic TextStyle, what if an error text is different from one component to another? Should we use the generic error text style, instead of just following the dsfr specification?

Reply:
Always follow the dsfr specifications, if an error text is different it will have a different name.

Reply:
but what if we saw mistakes in the specification? Do we made the same mistakes ? We should send them an email to tell theme (dude, this things right here, it's not a good idea)

Reply:
We have to consider the fact that because we are not the designers we will never be sure of what might be an error or not hence we have to trust the maintainers of the design system and adapt ourselves if it changes.

Reply:
I'm not designer but I feel like some part are just not well documented or wrong, but I see the point, let's follow the spec